site2/tools/publish-website.sh

```diff
@@ -65,7 +65,7 @@ PULSAR_SITE_TMP=/tmp/pulsar-site
       | grep -v $PULSAR_SITE_TMP/content/.htaccess \
       | xargs rm -rf
   fi
-  cp -r $GENERATED_SITE_DIR/content/ $PULSAR_SITE_TMP/content
+  cp -R $GENERATED_SITE_DIR/content/ $PULSAR_SITE_TMP/content
```

Comment:
The root cause should be that we change from `-r` to `-R` and the behavior changed.